### ISSUE TITLE
Does torch have any features or future plans to improve performance on ARM?

### ISSUE THREAD

**Issue by CLiqing** [pytorch/pytorch#128817]
### 🚀 The feature, motivation and pitch

Hi, I have faced some performance questions with ```torch``` recently. Our scenario is mainly on```ARM``` platform, but the same code often performs worse than on ```x86```. After analysis, we found that the performance  of one-core on ARM is weaker. The main advantage is that ARM has more CPU cores, but they are not fully utilized. ```Torch``` is more like a single-thread task.
So, do you have some features to use more cores for a better performance on ```ARM```? Or is there any future plan to improve performance and adapt to ```ARM```?

Another extra question, why does the backward need an extra thread? The current system does not seem to allow the forward and the backward to run in parallel.

cc @msaroufim @malfet @snadampal

**Comment by malfet** [comment]
Yes, we do looking at ARM performance and are making some improvements to it each release. For example half-precision matmuls will be much faster this release than the last one.
But issue right now is too generic and inactionable, i.e. from the issue description I can not tell which version of PyTorch are you using, how you've installed it and what particular ops you have performance issues with.

So closing this one, but please do not hesitate to open a new issue describing what operations/networks in your opinion exhibit this single-threaded performance problems compared to behavior on x86 platform
